h/config.py

```diff
@@ -43,6 +43,7 @@ def configure(environ=None, settings=None):
     settings_manager.set('es.client.timeout', 'ELASTICSEARCH_CLIENT_TIMEOUT', type_=float)
     settings_manager.set('es.url', 'ELASTICSEARCH_URL', required=True),
     settings_manager.set('es.index', 'ELASTICSEARCH_INDEX')
+    settings_manager.set('es.skip_plugin_check', 'ELASTICSEARCH_SKIP_PLUGIN_CHECK', type_=asbool)
```

Comment:
I'm thinking we should make this a positive param instead of a negative one to simplify the logic. By that I mean; es.check_icu_plugin. In which case the default would be True and the if statement below would be:
```
if es.check_icu_plugin:
```

Reply:
I don't anticipate us adding more plugins in the near future (especially since we just upgraded and didn't add any) so I think its ok to name it the specific plugin that it's checking for.